vcpkg.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
+  "name": "c-pal",
+  "version": "1.0.0",
```

Comment:
is there a c-pal package in vcpkg?

and it exists, who updates its version here?

Reply:
We're using a dummy package name. It's not like we're going to make a push to include a port in vcpkga for it. Not soon at least.
Fixed to be com-wrapper at least